content/reference/api/engine/_index.md

```diff
@@ -73,22 +73,22 @@ To see the highest version of the API your Docker daemon and client support, use
 ```console
 $ docker version
 Client: Docker Engine - Community
- Version:           28.3.0
+ Version:           28.3.1
  API version:       1.51
  Go version:        go1.24.4
  Git commit:        38b7060
- Built:             Tue Jun 24 15:44:12 2025
- OS/Arch:           linux/amd64
+ Built:             Wed Jul  2 20:56:31 2025
+ OS/Arch:           linux/arm64
  Context:           default
 
 Server: Docker Engine - Community
  Engine:
-  Version:          28.3.0
+  Version:          28.3.1
```

Comment:
Reminds me; we forgot to add 28.3 and the corresponding API version to the table further down; https://github.com/docker/docs/blob/main/content/reference/api/engine/_index.md#api-version-matrix